### ISSUE TITLE
Unable to load custom trained model

### ISSUE THREAD

**Issue by ektaarora3501** [ultralytics/yolov5#4112]
## ❔Question
Hey everyone!
I have a trained a custom object detection model using yolov5 for 4 classes. I have downloaded the best.pt file. I am still confused about how to load this model using pytorch. I tried running the following code to load the model as per Yolov5 official documentation ```model = torch.hub.load('ultralytics/yolov5', 'custom', path='best.pt')```  but when I tried printing model.names, I am not getting my custom class names. Is there anything I am missing? Thank you!

**Comment by glenn-jocher** [comment]
@ektaarora3501 pytorch hub models are wrapped in an AutoShape class, so you can access model names as 
```python
import torch

# Model
model = torch.hub.load('ultralytics/yolov5', 'yolov5s')
print(model.model.names)
```

**Comment by ektaarora3501** [comment]
Hey. Um I am still getting the default class names, I want my custom model to show the custom labels on which I trained it

**Comment by glenn-jocher** [comment]
@ektaarora3501 👋 hi, thanks for letting us know about this problem with YOLOv5 🚀. We've created a few short guidelines below to help users provide what we need in order to get started investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). Your code that reproduces the problem should be:

* ✅ **Minimal** – Use as little code as possible that still produces the same problem
* ✅ **Complete** – Provide **all** parts someone else needs to reproduce your problem in the question itself
* ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

In addition to the above requirements, for [Ultralytics](https://ultralytics.com/) to provide assistance your code should be:

* ✅ **Current** – Verify that your code is up-to-date with current GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been resolved by previous commits.
* ✅ **Unmodified** – Your problem must be reproducible without any modifications to the codebase in this repository. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all of the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) and providing a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by ektaarora3501** [comment]
Sure I will move this to bug section
